powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/model/UserInfoContext.java

```diff
@@ -20,18 +20,16 @@
 package io.getlime.security.powerauth.rest.api.spring.model;
 
 import io.getlime.security.powerauth.rest.api.model.entity.UserInfoStage;
-import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
-import lombok.NoArgsConstructor;
 
 /**
  * Class for passing the context attributes to user info provider.
  *
  * @author Petr Dvorak, petr@wultra.com
  */
 @Data
-@NoArgsConstructor
-@AllArgsConstructor
+@Builder
```

Comment:
FYI See `@SuperBuilder`, I know, it is experimental, but: 

> @SuperBuilder generates a protected constructor on the class that takes a builder instance as a parameter. This constructor sets the fields of the new instance to the values from the builder.

https://projectlombok.org/features/experimental/SuperBuilder